instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_trace.rb

```diff
@@ -98,11 +98,15 @@ def analyze_query(query:, &block)
 
           def execute_query(query:, &block)
             attributes = {}
-            attributes['graphql.operation.name'] = query.selected_operation_name if query.selected_operation_name
-            attributes['graphql.operation.type'] = query.selected_operation.operation_type
+            operation_type = query.selected_operation.operation_type
+            operation_name = query.selected_operation_name
+
+            attributes['graphql.operation.name'] = operation_name if operation_name
+            attributes['graphql.operation.type'] = operation_type
             attributes['graphql.document'] = query.query_string
 
-            tracer.in_span('graphql.execute_query', attributes: attributes, &block)
+            span_name = operation_name ? "#{operation_type} #{operation_name}" : operation_type
```

Comment:
This doesn't make too much sense at the spec level. OpenTelemetry consistently discourages high cardinality span names, and this by definition is unbounded cardinality.

Reply:
@robertlaurin Did you open an issue in the semantic-conventions repo about this?

Reply:
It does not seem like that came up in the PR reviews:

https://github.com/open-telemetry/opentelemetry-specification/pull/2456
https://github.com/open-telemetry/opentelemetry-specification/issues/1670

Reply:
@robertlaurin - Could you open an issue in the semantic-conventions repo about this?

Reply:
@karmingc, I apologize for how long the cycles on this PR have been. I brought this concern to the Semantic Conventions repo. 

Issue: https://github.com/open-telemetry/semantic-conventions/issues/1361
PR: https://github.com/open-telemetry/semantic-conventions/issues/1389

If you have any feedback on the proposed span name, let's discuss it in [the PR](https://github.com/open-telemetry/semantic-conventions/issues/1389). (cc @robertlaurin @arielvalentin)